frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/runLogs/LogsTab.tsx

```diff
@@ -72,6 +106,18 @@ const LogsTabForPodName: React.FC<{ podName: string }> = ({ podName }) => {
       .finally(() => setDownloading(false));
   };
 
+  const downloadDropdownItems: React.ReactElement[] = [];
+  if (podContainers.length > 1) {
+    downloadDropdownItems.push(
+      <DropdownItem key="current-container-logs" onClick={onDownload}>
+        Current step log
```

Comment:
@yih-wang Is this "current step log" or "current step logs"? I did this according to the current mocks
<img width="532" alt="Screenshot 2023-10-31 at 3 44 48 PM" src="https://github.com/opendatahub-io/odh-dashboard/assets/22885912/ada616cf-7577-415b-b62c-bcaa3beeef0d">

Reply:
@manaswinidas I don't see any change requirements in the content review history. But let us double check with @kaedward here. To clarify the case - one step could only have one log, which is what we are showing here in the log viewer. @kaedward Do you think it should be "current step log" or "current step logs"?

Reply:
BTW @manaswinidas I noticed that the toolbar looks a bit messy in the screenshot. Is this due to the side drawer being in a smaller size? If so, would you like some design guidance on how to make sure the component sizes are responsive and fit well?

Reply:
@yih-wang yes please, provide as much guidance, the toolbar is wrapping due to the width of the sidebar and we sorta just wrap as-is... it was fine when links had a little bit of whitespace, it's now colliding. We should definitely improve the look & feel.

Reply:
@yih-wang I think "Current step log" works here. 
Also, I think if we're pressed for space here, we could remove the text from the pause/resume button and display it on hover instead? Just an idea - but I'll leave the designing to the experts ;-)

Reply:
@manaswinidas @andrewballantyne Here is the guidance for the responsive state. Please let me know if anything is not clear, or didn't seem correct from the development perspective. I didn't provde such guidance a lot as this is usually covered by PatternFly components... So let me know if you need any further clarification.

**When to Switch to the Smaller Layout:**
The switch to the smaller size layout occurs when the screen size is smaller than 576px (defined as --pf-global--breakpoint--sm). In this responsive state, it's important to note that the side drawer occupies the entire screen when the screen size is smaller than 768px (--pf-global--breakpoint--md).

**Details of the Smaller Size Layout:**

1. **Step dropdown:** 
- Get rid of the label “Step” and only show the dropdown itself. 
- The width of the dropdown should be adjusted according to the screen/drawer width. Ensure the dropdown width precisely accommodates the row's width alongside other components in the refreshing state without causing any layout disruptions, considering that the refreshing state features the longest state label.
- If the text in the dropdown exceeds this width, truncate it. Users can access the full name through the dropdown menu.

2. **Status Labels:**
Shorten the status labels as follows:
<img width="505" alt="image" src="https://github.com/opendatahub-io/odh-dashboard/assets/23331885/59776c40-ca98-4eca-a578-09abfb2921b1">

3. **Download actions:**
Collapse the dropdown actions into a kebab menu. Note that the action labels will be changed to "Download..."

Here is the mock (showing the smallest screen example):

<img width="515" alt="image" src="https://github.com/opendatahub-io/odh-dashboard/assets/23331885/7e92f5e8-7761-4fac-b544-7a6b6d29a3ba">